pkg/storage/snapshot/BUILD.bazel

```diff
@@ -97,6 +97,7 @@ go_test(
         "//vendor/k8s.io/client-go/tools/cache:go_default_library",
         "//vendor/k8s.io/client-go/tools/cache/testing:go_default_library",
         "//vendor/k8s.io/client-go/tools/record:go_default_library",
+        "//vendor/k8s.io/utils/ptr:go_default_library",
```

Comment:
I see you have removed the used of this lib (thanks) the import still exists here, and in file tests/storage/BUILD.bazel need to update the deps so it will be removed
Other then that its great! ill approve once this is fixed